testing/sdk_cmd.py

```diff
@@ -110,9 +110,14 @@ def __call__(self, r):
 
     def _cluster_request():
         start = time.time()
-        response = requests.request(
-            method, url, auth=auth, verify=False, timeout=timeout_seconds, **kwargs
-        )
+
+        # check if we have verify key already exists.
+        if kwargs is not None and kwargs['verify'] is not None:
```

Comment:
Should we check have `kwarg.get("verify")` here? Otherwise we just end up with an exception with accessing `kwargs['verify']`?

Reply:
Im not a python ninja, but sure ill check that out, and make appropriate changes, oh and thank you for reviewing!